terraform/deployments/tfc-configuration/variables-production.tf

```diff
@@ -464,6 +464,23 @@ module "variable-set-rds-production" {
         has_read_replica             = true
       }
 
+      publisher = {
+        engine         = "postgres"
+        engine_version = "14"
```

Comment:
🤔  I wonder if we should go straight to 17, if it's a new database... Otherwise we're just storing ourselves up an upgrade task.